pkgs/top-level/python-packages.nix

```diff
@@ -8150,6 +8150,8 @@ in {
 
   solax = callPackage ../development/python-modules/solax { };
 
+  solc-select = callPackage ../development/python-modules/solc-select { };
```

Comment:
Is this a library or a cli tool? If the later we do not need an entry here.

Reply:
I removed it for now, can be readded if something depends on it